tests/test_utils.py

```diff
@@ -58,3 +60,46 @@ def test_upload_dir_to_s3(tmp_path):
                 call(tmp_path / 'subdir1/subdir3/bar.txt', 'myBucket', 'myPrefix/subdir1/subdir3/bar.txt'),
             ]
         )
+
+
+@mock_aws
+def test_list_files_in_s3(s3_bucket):
+    prefix = 'geotiffs'
+    geotiffs = [
+        f'{prefix}/my-file1.tif',
+        f'{prefix}/my-file2.tif',
+        f'{prefix}/my-file3.tif',
+    ]
+
+    for tif in geotiffs:
+        ut.S3_CLIENT.put_object(Bucket=s3_bucket, Key=tif)
+
+    files = ut.list_files_in_s3(s3_bucket, prefix)
+
+    assert len(files) == len(geotiffs)
+    assert files[0]['Key'].endswith('.tif')
+
+
+@mock_aws
+def test_download_file_from_s3(tmp_path, s3_bucket):
+    download_path = 'geotiffs/my-file.tif'
+    dest_dir = Path(tmp_path)
+
+    ut.S3_CLIENT.put_object(Bucket=s3_bucket, Key=download_path)
+
+    output_path = ut.download_file_from_s3(s3_bucket, download_path, dest_dir)
+
+    assert output_path == tmp_path / 'my-file.tif'
+
+
+@pytest.fixture
+def s3_bucket(scope='function'):
```

Comment:
I don't know much about pytest fixtures, but should this be an argument of the decorator, as shown [here](https://docs.pytest.org/en/stable/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session)? Also, looks like [`function` is already the default](https://docs.pytest.org/en/stable/how-to/fixtures.html#fixture-scopes), so this can probably just be removed.
```suggestion
@pytest.fixture
def s3_bucket():
```